lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +12,11 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    tokens = [symbol for symbol in text if symbol.isalpha()]
```

Comment:
return as soon as posible. Do not create the variable

Reply:
done